### ISSUE TITLE
Broken links in PDF

### ISSUE THREAD

**Issue by blmoore** [blmoore/md-cv#1]
Inconsistent behavious with links in the PDF version. Most mailto's seem to work but http less so, at least with OSX preview.

**Comment by ZuzooVn** [comment]
Is it ok now?

**Comment by blmoore** [comment]
Not really, only the `mailto` links work. As to why this is: ¯\\\_(ツ)_/¯

**Comment by ZuzooVn** [comment]
Open the HTLM file from Chrome and Print it as PDF. Everything will be OK
